src/libraries/tests.proj

```diff
@@ -17,7 +17,7 @@
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Transactions.Local\tests\System.Transactions.Local.Tests.csproj" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(LinkerTests)' != 'true'">
```

Comment:
How would I build both linker tests and normal tests? Unrelated but similar, how would I build tests projects for all configurations but not run the packaging test but the normal tests? I think it would make sense to condition the three different testing modes on corresponding flags, ie `TestLinker`, `TestPackages`, `TestAssemblies`.

Reply:
good point, I'll add this.